### PR TITLE
Gcp multinet

### PR DESCRIPTION
**Pull request by radicleart**
Adds the ability to switch between mainnet, testnet and devnet via query string parameter;

```
net=mainnet or nothing for mainnet
net=testnet for testnet
net=devnet for devnet
```

See also [PR 17](https://github.com/Trust-Machines/sbtc-bridge-api/pull/17) on the API which support this change.

Note not all values are fully supported / tested.

### CODE REVIEW

**Comment by igorsyl** [comment]
I think stacks-api calls `net` `chain`

File: src/lib/config.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    VITE_SBTC_CONTRACT_ID: 'ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.sbtc-alpha',
+    VITE_BRIDGE_API: 'http://localhost:3010/bridge-api/v1',
+    VITE_STACKS_API: 'http://localhost:3999',
+    VITE_STACKS_EXPLORER: 'http://localhost:8000',
+    VITE_BSTREAM_EXPLORER: 'https://mempool.space/testnet',
+    VITE_MEMPOOL_EXPLORER: 'https://mempool.space/testnet/api',
+    VITE_BLOCKCYPHER_EXPLORER: 'https://api.blockcypher.com/v1/btc/test3',
+}
+
+export let CONFIG = MAINNET_CONFIG;
```
Comment (sergey-shandar) [inline src/lib/config.ts:48]: Awesome!

**Review by sergey-shandar — CHANGES_REQUESTED** [review]
LGTM, I like that we moved config from env var.

@radicleart please fix unit tests.

**Comment by radicleart** [comment]
Fixed copy error: https://github.com/Trust-Machines/sbtc-bridge-web/issues/89

**Comment by radicleart** [comment]
> LGTM, I like that we moved config from env var.
> 
> @radicleart please fix unit tests.

Fixed.

**Comment by radicleart** [comment]
Just noticed this in package.json of the client application

```
	"scripts": {
		"docker": "docker build -t sbtc-bridge-web . && docker run -p 8080:8080 sbtc-bridge-web",
		"start": "node server.js",
```

and a dependency on express?

@igorsyl  is there any need for this any more - how it would be used ?

**Review by sergey-shandar — APPROVED** [review]

**Comment by igorsyl** [comment]
> 

I will remove dep on express. Thanks!

File: src/lib/config.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+const TESTNET_CONFIG = {
+    VITE_PUBLIC_APP_NAME: 'sBTC Bridge Testnet',
+    VITE_PUBLIC_APP_VERSION: '1.0.0',
+    VITE_ORIGIN: 'https://sbtc.world',
+    VITE_NETWORK: 'testnet',
+    VITE_SBTC_WALLET: 'tb1q6zlpyrzvzjcrf6dlsctcrh9yl3dwfktsw0nclq',
+    VITE_SBTC_CONTRACT_ID: 'ST3N4AJFZZYC4BK99H53XP8KDGXFGQ2PRSPNET8TN.sky-blue-elephant',
+    VITE_BRIDGE_API: 'http://localhost:3010/bridge-api/v1',
+    VITE_STACKS_API: 'https://api.testnet.hiro.so',
+    VITE_STACKS_EXPLORER: 'https://explorer.hiro.so',
```
Comment (igorsyl) [inline src/lib/config.ts:12]: https://explorer.hiro.so/?chain=testnet

Comment (radicleart) [reply src/lib/config.ts:12]: > https://explorer.hiro.so/?chain=testnet

we append this on check of CONFIG.network.

**Comment by radicleart** [comment]
Closing this PR - absorbed by [PR 94](https://github.com/Trust-Machines/sbtc-bridge-web/pull/94)

**Pull request closed without merging**